products/jira.md

```diff
@@ -152,6 +200,62 @@ releases:
     link:
       https://confluence.atlassian.com/jirasoftware/jira-software-8-13-x-release-notes-1018783360.html
 
+-   releaseCycle: "8.12"
+    releaseDate: 2020-08-25
+    eol: 2022-08-26
+    latest: "8.12.3"
+    latestReleaseDate: 2020-10-06
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-12-3-1026523933.html
+
+-   releaseCycle: "8.11"
+    releaseDate: 2020-07-14
+    eol: 2022-07-15
+    latest: "2020-08-16"
+    latestReleaseDate: 
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-11-1-1018767316.html
+
+-   releaseCycle: "8.10"
+    releaseDate: 2020-06-22
+    eol: 2022-06-23
+    latest: "8.10.1"
+    latestReleaseDate: 2020-06-22
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-10-1-1013852224.html
+
+-   releaseCycle: "8.9"
+    releaseDate: 2020-05-18
+    eol: 2022-05-20
+    latest: "8.9.1"
+    latestReleaseDate: 2020-06-15
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-9-1-1005349334.html
+
+-   releaseCycle: "8.8"
+    releaseDate: 2020-03-18
+    eol: 2022-03-19
+    latest: "8.8.1"
+    latestReleaseDate: 2020-04-21
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-8-1-1003521744.html
+
+-   releaseCycle: "8.7"
+    releaseDate: 2020-02-02
+    eol: 2022-02-03
+    latest: "8.7.1"
+    latestReleaseDate: 2020-02-09
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-7-1-994312443.html
+
+-   releaseCycle: "8.6"
+    releaseDate: 2019-12-16
+    eol: 2022-02-03
```

Comment:
```suggestion
    eol: 2021-12-17
```

According to https://web.archive.org/web/20211117125223/https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html.